k8s/helm/charts/api/templates/hpa.yaml

```diff
@@ -36,7 +36,7 @@ spec:
       object:
         describedObject:
           kind: Service
-          name: accounts
+          name: api
```

Comment:
@AliakseiMalyshau is this ok?

Reply:
@NikolayRn yes, it was a mistake from my side, for RPS scaling should be specified the right service name. For API service name is `api`, not `accounts`.

Reply:
You can check the others HPA's, I just copied an example from `accounts` and forgot to change the service name.